internal/core/items.rs

```diff
@@ -331,6 +332,10 @@ pub struct BorderRectangle {
     pub background: Property<Brush>,
     pub border_width: Property<LogicalLength>,
     pub border_radius: Property<LogicalLength>,
+    pub border_top_left_radius: Property<LogicalLength>,
+    pub border_top_right_radius: Property<LogicalLength>,
+    pub border_bottom_left_radius: Property<LogicalLength>,
+    pub border_bottom_right_radius: Property<LogicalLength>,
```

Comment:
This is a lot of properties in the Rectangle for the runtime.
Ideally, this could be only one property like
`pub border_radius: Property<LogicalBorderRadius>,`
It is fine if the runtime propery don't exactly match the slint properties. Another example is Path::command
The way to implement that is to have it as an Expression of a struct that eventually get lowered to LogicalBorderRadius  (like PathElement or GridLayoutCellData or PropertyAnimation or StateInfo) as some example.

Although this can totally be done in another PR,  4 property is not the end of the world.

But I believe however you can already remove the `border_radius` property now since it's replaced by independent properties.

Reply:
Is it possible to remove the `border_radius` property with this kind of bindings, for example?

```
Rectangle {
    border-radius: root.border-radius;
    border-top-right-radius: 2 * self.border-radius;
    border-bottom-right-radius: 2 * self.border-radius;
}
```

Reply:
I think that's fine because the compiler can re-create (materialize) the property if needed